docs/developers/docker.rst

```diff
@@ -61,6 +61,29 @@ data, run:
 
     docker-compose -f docker-compose-dev.yml rm -f
 
+Interactive sessions
+--------------------
+
+For interactive sessions inside the container simply attach a shell to the
+running instance:
+
+.. code-block:: shell
+
+    docker exec -it invenio_web_1 bash
+
+.. note::
+    Some tools (mostly ncurses-based ones) require the ability to detect the
+    used terminal. To enable this, set the `TERM` environment variable to the
+    correct value as a first command inside the container, e.g. by running:
+
+    .. code-block:: shell
+
+        export TERM=xterm
```

Comment:
By default TERM is not defined inside the interactive session (bash inside the container). So e.g. `bibsched` won't even start because curses is not able to detect the terminal. The variable is only for the interactive session and is only populates to child progresses of bash, not to other processes inside the container or (especially) to professes outside the container.